<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPaket</class>
 <widget class="QWidget" name="FormPaket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>160</width>
     <height>110</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="kodePaketLabel">
      <property name="text">
       <string>Kode Paket</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="kodePaketComboBox">
      <item>
       <property name="text">
        <string>1 Kg</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2 Kg</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4 Kg</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6 Kg</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="namaPaketLabel">
      <property name="text">
       <string>Nama Paket</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="namaPaketLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="hargaLabel">
      <property name="text">
       <string>Harga</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="hargaLineEdit"/>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
